test/app_integration/services/supertest.js

```diff
@@ -0,0 +1,9 @@
+import { format as formatUrl } from 'url';
+
+import supertestAsPromised from 'supertest-as-promised';
```

Comment:
<img width="750" alt="screen shot 2017-10-12 at 14 57 43" src="https://user-images.githubusercontent.com/5259/31496985-baf21e66-af5d-11e7-978e-384ae6e26eaa.png">

https://github.com/WhoopInc/supertest-as-promised

Reply:
This was just copied over directly from the api_integration tests, so I'd prefer to keep it for now.  We should upgrade our dependencies and get both suites fixed up together.

Reply:
It's not super-obvious when looking at the deps list that this one is deprecated. If you're not doing it in this PR, can you create an issue for this?

Reply:
@kjbekkelund I'm working on removing supertest now, so I'll follow up with a PR